packages/pwa/app/hooks/use-add-to-cart-modal.js

```diff
@@ -165,7 +166,7 @@ export const AddToCartModal = () => {
                                     {productSubTotal &&
                                         intl.formatNumber(productSubTotal, {
                                             style: 'currency',
-                                            currency: currency || DEFAULT_CURRENCY
+                                            currency: currency || site.defaultCurrency
```

Comment:
```suggestion
                                            currency: currency || site.l10n.defaultCurrency
```

Reply:
I've removed the use of defaultCurrency as it appears that currency in this hook will always be defined. See the explanation https://github.com/SalesforceCommerceCloud/pwa-kit/pull/391#discussion_r823171565